setup-elytron-guides/setup_elytron_linux.adoc

```diff
@@ -0,0 +1,148 @@
+---
+layout: setup-elytron
+title: Setting up WildFly Elytron on Linux & WSL
+permalink: /guides/setup-elytron-linux/
+os: linux
+---
+:imagesdir: ../../assets/images/setup-elytron/
+
+:toc: macro
+:toc-title:
+
+This page will guide you through the steps needed to setup a development environment for WildFly Elytron on Linux, or the https://docs.microsoft.com/en-us/windows/wsl/about[Windows Subsystem for Linux (WSL)]. For WSL developers, all commands should be entered into the WSL terminal _(i.e. not Command Prompt, PowerShell, or Git Bash)_.
+
+toc::[]
+
+== Installing Java
+
+. First, let's check if you already have Java installed.
+    .. Open a terminal, and run `java --version`.
```

Comment:
s/terminal,/terminal (sorry missed some small grammar things when I was reviewing the google doc versions)